tests/ignite/handlers/test_checkpoint.py

```diff
@@ -592,6 +593,31 @@ def _test_existance(atomic, _to_save, expected):
     _test_existance(atomic=True, _to_save=to_save_non_serializable, expected=False)
 
 
+@pytest.mark.skipif(
+    parse_version(torch.__version__) < parse_version("1.4.0"), reason="Zipfile serialization was introduced in 1.4.0"
+)
+def test_disk_saver_zipfile_serialization_keyword(dirname):
+    model = DummyModel()
+    to_save = {"model": model}
+
+    saver = DiskSaver(dirname, create_dir=False, _use_new_zipfile_serialization=False)
+    fname = "test.pt"
+    saver(to_save, fname)
+    fp = os.path.join(saver.dirname, fname)
+    assert os.path.exists(fp)
+    saver.remove(fname)
+
+
+@pytest.mark.xfail
```

Comment:
Can't we use
```python
with pytest.raises(TypeError, match=r"error message to match"):
    saver(to_save, fname)
```

Reply:
Sure, I already had the hunch there was an alternative to xfail being used..